common.lic

```diff
@@ -192,8 +192,14 @@ module DRC
 
   def stow_hands
     pause 1
-    bput('stow right', 'You put') if checkright
-    bput('stow left', 'You put') if checkleft
+    stow_hand('right') if checkright
+    stow_hand('left') if checkleft
+  end
+
+  def stow_hand(hand)
+    braid_regex = /The braided (.+) is too long/
+    result = bput("stow #{hand}", 'You put', braid_regex)
+    DRCI.dispose_trash(DRC.get_noun(Regexp.last_match(1))) if braid_regex.match(result)
```

Comment:
The `DRCI.dispose_trash` is still necessary. I was referring to `DRC.get_noun`. Try removing `DRC` and let me know if that works.